planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -184,7 +184,6 @@ void PullOverModule::onExit()
   steering_factor_interface_ptr_->clearSteeringFactors();
 
   // A child node must never return IDLE
-  // https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/include/behaviortree_cpp_v3/basic_types.h#L34
```

Comment:
the link ia already invalid now. 
move to https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/include/behaviortree_cpp/basic_types.h#L35, but remove it considering the maintenance cost